### PR TITLE
[v7r3] SiteDirector: do not send dirac-install if it's py3 pilot

### PR DESCRIPTION
**Pull request by fstagni**
This waits for `Pilot/devel` branch to merge to `Pilot/master` branch (after https://github.com/DIRACGrid/Pilot/pull/133)

BEGINRELEASENOTES

*WMS
CHANGE: SiteDirector: do not send dirac-install if it's py3 pilot

ENDRELEASENOTES

### CODE REVIEW

**Comment by fstagni** [comment]
review, please

File: .github/workflows/pilotWrapper.yml [hunk 1]
```diff
@@ -11,10 +11,10 @@ jobs:
       fail-fast: False
       matrix:
         python:
-          - 2.6.9
```
Comment (chaen) [inline .github/workflows/pilotWrapper.yml:14]: isn't the pilot still supposed to be able to run on 2.6 ?

Comment (fstagni) [reply .github/workflows/pilotWrapper.yml:14]: Well, 2.6 was for SLC6, which I believe that by now is finally gone (we hopefully won't find anymore Worker Nodes with SLC6, and even more by the time this go to production).

Comment (chaen) [reply .github/workflows/pilotWrapper.yml:14]: I wouldn't bet on that, but okay :-)

**Review by chaen — APPROVED** [review]

**Pull request merged**